README.md

```diff
@@ -23,11 +23,13 @@ isLocal("184.55.123.2"); // false
 # Valid Private IP'S
 * `::`
 * `::1`
-* `fe80::1`
+* `fe80::/10`
+* `fc00::/7`
 * `10.0.0.0` - `10.255.255.255`
 * `127.0.0.0` - `127.255.255.255`
 * `172.16.0.0` - `172.31.255.255`
 * `192.168.0.0` - `192.168.255.255`
+* `169.254.1.0` - `169.254.254.255`
```

Comment:
Could you sort this numerically? (It is killing me for some reason :p).

IE: 169 before 172 and fc00 before fe80

Reply:
I will do it tomorrow!